### ISSUE TITLE
support KeeShare

### ISSUE THREAD

**Issue by basbebe** [keepassium/KeePassium#138]
**Describe a feature you'd like**
I'd like KeeShare files fully supported.  

> KeeShare allows you to share a subset of your credentials with others and vice versa.

see
- https://www.tpidg.us/keeshare-secure-password-sharing-nsc-78/
- https://github.com/keepassxreboot/keepassxc/blob/develop/docs/topics/KeeShare.adoc
- 
At the moment it works, but only if the Keeshare files are created without a signature.
Also I don't know if all features like sync, read-only, etc. would be fully supported as of right now.

**Additional context**
It makes it more easy to have a more granulated access policy.  
e.g. have a main keepassxc database and share only a keeshare file w/ e.g. web logins to share (e.g. via syncthing) with Strongbox on iOS.

**Comment by keepassium** [comment]
This is awesome! I did not know about KeeShare, but it seems like a great workaround for multi-user access.

Implementing it would require a lot of ground work, but seems very much worth it.

Thank you for the suggestion!

**Comment by cdrehner** [comment]
Is this great feature on the roadmap for future releases?

**Comment by keepassium** [comment]
> Is this great feature on the roadmap for future releases?

Definitely yes.

**Comment by chriswayg** [comment]
I would love to see support added for KeeShare as well. Currently KeeShare groups are shown and are initially synchronized, but data will quickly get out of sync in various ways, if you actually try to edit or create items inside the shared group from within KeePassium. Here is what I tested:

### KeeShare setting
- Synchronize on all

### User A
- iPhone user A using KeePassium
- iPhone user A using Strongbox
- macOS user A using KeePassXC

### User C
- iPhone user C using KeePassium
- iPhone user C using Strongbox
- macOS user C using KeePassXC

### Test sync behavior

- All devices synced

- Create new item in shared Group by macOS user A using KeePassXC
	- synced correctly to...
		- iPhone user A using KeePassium (db initially refused to refresh - requiring restart of KeePassium)
		- iPhone user A using Strongbox
		- macOS user C using KeePassXC
		- iPhone user C using Strongbox
		- iPhone user C using KeePassium (db initially refused to refresh)

- Create a new item in shared Group by macOS user A using KeePassium
	- does not show up for...
		- iPhone user C using KeePassium
		- iPhone user C using Strongbox
		- macOS user C using KeePassXC
		
Basically items become private to the KeePassium user who created them in the shared group. Editing existing items inside the shared group creates quite a mess rather quickly... -  This makes using the KeeShare feature on KeePassXC unusable, if you are also using any KeePass apps (such as KeePassium) on devices that do not yet support KeeShare.

**Comment by rainforest1155** [comment]
I'm currently using Kypass on iOS and was looking around what other alternatives there are. I do already use Keepassium free for a single database on a USB stick. On my main database, I also make use of the KeeShare feature, but it seems that this isn't supported by any of the iOS apps I looked at.

If this were to become available on KeePassium, it would make the decision easier for me as I do like the perpetual premium offer - that's a really great idea as I'm generally not a fan of subscriptions.

**Comment by glciampaglia** [comment]
I am a Linux/Android user and do not use Keepassium, but I am trying to help my wife migrate all her passwords to Keepass format. I was hoping to make use KeeShare for passwords of shared accounts, but since she uses an iPhone we are in the same situation of the previous poster who mentioned not there being any iOS apps with KeeShare support. Having KeeShare support in Keepassium would be a big plus for a use case like ours.

**Comment by thiemjoerg** [comment]
Hi,
when will this feature be available? My wife and me are in the same situation :-(
Regards
